webserver/src/test/java/org/jboss/test/arquillian/ce/webserver/WebServerTomcat7BasicSecureTest.java

```diff
@@ -23,25 +23,31 @@
 
 package org.jboss.test.arquillian.ce.webserver;
 
+import java.net.URL;
+
+import javax.websocket.ClientEndpoint;
+
+import org.jboss.arquillian.ce.api.OpenShiftResource;
+import org.jboss.arquillian.ce.api.OpenShiftResources;
 import org.jboss.arquillian.ce.api.Template;
 import org.jboss.arquillian.ce.cube.RouteURL;
 import org.jboss.arquillian.container.test.api.RunAsClient;
 import org.jboss.arquillian.junit.Arquillian;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
-import javax.websocket.ClientEndpoint;
-import java.net.URL;
-
 /**
  * @author fspolti
  */
 @RunWith(Arquillian.class)
 @Template(url = "https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/webserver/jws30-tomcat7-https-s2i.json",
         labels = "application=jws-app"
 )
+@OpenShiftResources({
+    @OpenShiftResource("https://raw.githubusercontent.com/${template.repository:jboss-openshift}/application-templates/${template.branch:master}/secrets/jws-app-secret.json")
```

Comment:
In this occurence, we are using single indent. In several others (in the project and this PR), we are using double indent. This is a very minor, but we should standardize and use in a single way from now on. I propose single indent, since it's the convention for the language.
This is not a blocker to merge, but instead something we should have in mind and start adopting a single pattern.